llvm/test/CodeGen/AMDGPU/GlobalISel/buffer-load-store-pointers.ll

```diff
@@ -0,0 +1,71 @@
+; RUN: not --crash llc -global-isel -mtriple=amdgcn -mcpu=gfx90a -verify-machineinstrs -stop-after=instruction-select < %s
```

Comment:
Generally don't precommit tests that fail. Also don't need -verify-machineinstrs

Reply:
Good to know about not precommiting tests that crash - I remember way back in the day someone wanting a `not --crash` test for a bug I was fixing, hence the precommit

Reply:
Sometimes it makes sense, but I don't think it does here